README.rst

```diff
@@ -58,153 +58,30 @@ Using with Content Libraries on a Tutor Devstack
-
-#. Now you should be able to use Blockstore in Studio.
-
-   To edit Blockstore content libraries in Studio, you'll need to install either `the Content Libraries v2 Frontend <https://github.com/openedx/frontend-app-library-authoring/>`_ or `Ramshackle <https://github.com/open-craft/ramshackle/>`_. Alternatively, you can use the `Content Libraries v2 REST API <https://github.com/openedx/edx-platform/blob/master/openedx/core/djangoapps/content_libraries/urls.py>`_ to create content programmatically.
-
-   To use Blockstore library content in a course, open your course in Studio. In its advanced settings, add ``library_sourced`` to the list of "advanced block types". In the "Unit Edit View" in Studio, find the green "Add New Component" buttons. Click Advanced > Library Sourced Content. Edit the new Library Sourced Content XBlock to enter the XBlock ID of the library content that you'd like to use. It should be similar to ``lb:DeveloperInc:MyLibrary:1`` (note: ``lb:`` is short for "Library Block" and should not be confused with the ``lib:`` prefix used to identify a library).
-
-#. Optional: To log in to Blockstore in your web browser directly, you'll need to configure SSO with your devstack. Most people won't need to do this, but it's helpful for debugging or development.
-
-   #. Go to http://localhost:18000/admin/oauth2_provider/application/ and add a new application
-   #. Set "Client id" to ``blockstore-sso-key``
-   #. Set "Redirect uris" to ``http://localhost:18250/complete/edx-oauth2/``
-   #. Set "Client type" to "Confidential"
-   #. Set "Authorization grant type" to "Authorization code"
-   #. Set "Name" to ``blockstore-sso``
-   #. Check "Skip authorization"
-   #. Press "Save and continue editing"
-   #. Go to http://localhost:18000/admin/oauth_dispatch/applicationaccess/
-   #. Click "Add Application Access +", choose Application: ``blockstore-sso`` and set Scopes to ``user_id``, then hit "Save"
-   #. Copy ``blockstore/settings/private.py.example`` to ``blockstore/settings/private.py``
-   #. In ``private.py``, set ``SOCIAL_AUTH_EDX_OAUTH2_SECRET`` to the random "Client secret" value.
-   #. Now you can login at http://localhost:18250/login/
-
-#. Optional: If running an Open edX devstack under a project name different
-   than the default (support for which was introduced
-   [here](https://github.com/openedx/devstack/pull/532)), simply export
-   ``OPENEDX_PROJECT_NAME`` and substitute the container names in the commands
-   above accordingly.
-
--------------
-Running Tests
--------------
-
-Unit Tests
-----------
-
-To run the unit tests, get into the blockstore container:
-
-.. code::
-
-  make blockstore-shell
-
-
-And then run:
-
-.. code::
-
-  make test
-
-To save on overhead while running individual tests, from within the container, you can do:
-
-
-.. code::
-
-  DJANGO_SETTINGS_MODULE=blockstore.settings.test ./manage.py test dotted.path.To.test
-
```

Comment:
We can keep this section -- we still need the blockstore docker container and these instructions for running tests.